google/ggrc-core
# GGRC-6756 Properly calculate predefined values for issue tracker dict during import

# Issue description

Predefined values for issue tracker are incorrectly calculated for Issue tracked objects.

For assessment, values have to be fetched in the following order:
1. Values in Assessment template, if assessment template specified and issue tracker values are available
2.  Values from Audit if issue tracker values are available for audit
3. Default values

For assessment template, values have to be fetched in the following order:
1.  Values from Audit if issue tracker values are available for audit
2. Default values

For Issue issue tracker values have to be fetched from default values.

Moreover, values have to be fetched in that order even if import file doesn't contain corresponding columns!

# Steps to test the changes

1. Prepare import spreadsheet and import it.
2. Ensure that issue tracker values of imported objects are set to correct values

# Sanity checklist




# PR Review checklist
